egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -16,8 +16,12 @@ decode_dir_affix=
 num_epochs=4
 remove_egs=false
 common_egs_dir=
-
 affix=
+
+# decoding options
+extra_left_context=50
```

Comment:
I would suggest actually using left_context to set this value

`if [ -z $extra_left_context ]; then
   extra_left_context=`echo $left_context + 10|bc`
fi`